src/main/java/cc/polyfrost/oneconfig/config/elements/BasicOption.java

```diff
@@ -134,7 +134,8 @@ public void keyTyped(char key, int keyCode) {
 
     public void drawDescription(long vg, int x, int y, InputHandler inputHandler) {
         if (description.trim().equals("")) return;
-        boolean hovered = inputHandler.isAreaHovered(getNameX(x), y, RenderManager.getTextWidth(vg, name, 14f, Fonts.MEDIUM), 32f);
+        final NanoVGHelper nanoVGHelper = NanoVGHelper.INSTANCE;
```

Comment:
why not at this point just make it so this is non static so you do vg = new NanoVGHelper(); and then all the methods are vg.drawRect() without the vg parameter as this would be part of the helper instance